test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.util.TypicalPersons;
+
+import static org.junit.Assert.*;
+
+public class SortCommandTest {
+    private final AddressBook addressBook = new TypicalPersons().getTypicalAddressBook();
+    private final TypicalPersons td = new TypicalPersons();
+
+    @Test
+    public void testSortCommand() throws Exception {
```

Comment:
does this throw exception?

Reply:
Noted, and updated. It's supposed to catch the exception and inform us with a message, instead of just throwing it.